mobile/android/android-device-file-system.ts

```diff
@@ -23,12 +24,35 @@ export class AndroidDeviceFileSystem implements Mobile.IDeviceFileSystem {
 		return this.adb.executeShellCommand(listCommandArgs);
 	}
 
-	public getFile(deviceFilePath: string): IFuture<void> {
-		return Future.fromResult();
+	public getFile(deviceFilePath: string, outputPath?: string): IFuture<void> {
+		return (() => {
+			let stdout = false;
+			if (!outputPath) {
+				let fileName = path.basename(deviceFilePath);
+				outputPath = path.join(shelljs.tempdir(), this.$fs.getUniqueFileName(fileName));
```

Comment:
In case you use `temp.path()` in combination with `temp.track()`, the downloaded file will be automatically removed after the process termination. This way users's temp directories will be cleaned from our files.